.gitignore

```
@@ -5,3 +5,5 @@
 /*.iml
 /.idea
 /logs/
+/test/bench/lib/*.pyc
+/test/bench/cloudstone*/*.pyc
```

This might be better as:
```suggestion
*.pyc
```